pvlib/scaling.py

```diff
@@ -185,7 +191,7 @@ def _compute_wavelet(clearsky_index, dt=None):
     Model (WVM) for Solar PV Power Plants. IEEE Transactions on Sustainable
     Energy, vol. 4, no. 2, pp. 501-509, 2013.
 
-    [3] Wavelet Variability Model - Matlab Code:
+    [2] Wavelet Variability Model - Matlab Code:
     https://pvpmc.sandia.gov/applications/wavelet-variability-model/
```

Comment:
```suggestion
    .. [2] Wavelet Variability Model - Matlab Code:
       https://pvpmc.sandia.gov/applications/wavelet-variability-model/
```

Could you edit the format for [1] as well?  Two leading dots before [1], and align the following lines with the "]" character.

Reply:
Since that same reference is in the other functions, I updated all of those as well.